docs/job-scheduling.md

```diff
@@ -53,7 +53,11 @@ Resource allocation can be configured as follows, based on the cluster type:
   on the cluster (`spark.executor.instances` as configuration property), while `--executor-memory`
   (`spark.executor.memory` configuration property) and `--executor-cores` (`spark.executor.cores` configuration
   property) control the resources per executor. For more information, see the
-  [YARN Spark Properties](running-on-yarn.html).
+  [YARN Spark Properties](running-on-yarn.html#spark-properties).
```

Comment:
This line change is irrelevant to K8s.

Reply:
Yes.
Because the link is not accurate even if it was not related to K8S.
Do you want me revert it?

Reply:
reverted.